app/js/lib/reducers/form_content.js

```diff
@@ -5,9 +5,31 @@ var objectAssign = require( 'object-assign' ),
 	initialState = {
 		amount: 0,
 		isCustomAmount: false,
-		paymentType: 'BEZ'
+		paymentType: 'BEZ',
+		recurringPayment: 0, // 0 = one-off, 1 = recurring Payment
+		paymentPeriod: 0 // 0, 1, 3, 6 or 12
```

Comment:
You're right, using one field instead of two will get rid of that rule stuff and its tests and still achieve the same effect. I'll change this PR.